admin/build.sbt

```diff
@@ -0,0 +1,46 @@
+lazy val piezoAdmin = (project in file(".")).enablePlugins(PlayScala)
+
+bashScriptExtraDefines ++= Seq(
+  "mkdir -m 664 /var/run/piezo-admin/",
```

Comment:
These commands appear in the .deb install script. (Or .rpm, etc., but we only build .debs.)
So you will already be root.
Alternatively, you could put the pid in /tmp, though /var/run is a more standard location.